src/pages/Home/home.scss

```diff
@@ -0,0 +1,154 @@
+@import "../../common.scss";
+
+.home-container {
+  background-color: $secondaryColor;
+  position: relative;
+  color: $font-primary;
+
+  .top {
+    display: flex;
+    gap: 24px;
+    align-items: center;
+    padding: 10px;
+    min-height: 20vh;
+    box-shadow: $box-shadow;
+    margin-bottom: 25px;
+
+    .left {
+      flex: 1;
+      justify-self: end;
+
+      img {
+        float: right;
+        width: $img-width;
+        height: $img-height;
+
+        @include mobile {
+          width: 100%;
+          height: 100%;
+          float: none;
+        }
+      }
+    }
+
+    .right {
+      flex: 1;
+      align-self: start;
+      text-align: right;
+      padding: 20px;
+      display: flex;
+      flex-direction: column;
+      gap: 12px;
+
+      p {
+        font-size: 20px;
+        font-weight: 400;
+
+        @include mobile {
+          font-size: 13px;
+        }
+      }
+    }
+  }
+
+  .bottom {
+    display: flex;
+    flex-direction: column;
+
+    @include mobile {
+      background-color: $primaryColor;
+    }
+
+    h2 {
+      text-align: center;
+      text-transform: capitalize;
+    }
+
+    .search {
+      display: flex;
+      align-items: center;
+      margin: 24px auto;
+      width: 60vw;
+      padding: 10px;
+      border-radius: 50px;
+      background-color: $search-bg;
+
+      @include mobile {
+        width: 40%;
+        padding: 5px;
+        height: 45px;
+      }
+
+      input {
+        padding: 5px;
+        outline: none;
+        border: none;
+        width: 100%;
+        font-size: 25px;
+        font-weight: 400;
+        background-color: transparent;
+      }
+    }
+
+    .items {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      list-style: none;
+      padding: 5px;
+
+      @include xs {
+        margin: 10px;
+        font-size: 18px;
+        grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
+      }
+
+      .item {
+        display: flex;
+        justify-content: space-between;
+
+        @include mobile {
+          display: flex;
+          flex-direction: column-reverse;
+          font-size: 18px;
+          padding: 2px;
+          justify-content: flex-end;
+          align-items: flex-end;
+          row-gap: 44px;
+        }
+
+        @include xs {
+          font-size: 12px;
+          padding: 5px;
+        }
+
+        font-size: 20px;
+        border: 2px solid rgba(153, 153, 153, 0.13);
+        padding: 10px;
+        margin: 3px;
+
+        &:hover {
+          transform: scaleX(0.97);
+        }
+
+        .item-container {
+          @include mobile {
+            padding: 6px;
+            text-align: right;
+            margin: 2px;
+          }
+
+          padding: 15px;
+          margin: 5px;
+          text-align: left;
+
+          h4,
+          h3 {
+            color: $font-primary;
+            font-weight: 400;
+            margin-bottom: 5px;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
I have realized that you might not have fully understood the previously requested changes I will try to elaborate to you as best as I can.

- [ ]  The theme that you have chosen for the project is a dark-blue-ish color theme which does not match i any way the color schemes that are required for this project. You have two themes that you can choose from one being a pink-ish color theme and the other one is a sky-blue-ish color theme. The requested changes by the [previous reviewer](https://github.com/melashu/livecrypto/pull/1#discussion_r996214252) echoed this to you. Kindly change the color theme that you chose for your design and kindly use one of the two provided color theme alternatives. This way, your project will match the Behance design and your project will be ready for approval.

**This is one of the desired themes for this project:**

![image](https://user-images.githubusercontent.com/86318284/195975961-03facc4a-ef26-433c-aff0-0e25200931d7.png)

**This is the other theme that you can choose from:**

![image](https://user-images.githubusercontent.com/86318284/195976141-d3ba1f7d-68ef-4893-9075-2e8f1f6d0b0c.png)

**Now here is your design currently:**

![image](https://user-images.githubusercontent.com/86318284/195976158-c63534f8-e18b-46ef-9f87-99c3d5b89346.png)

**Note the contrast?** Kindly ensure that you make this change and your project will be ready for approval.